core/Makefile

```diff
@@ -46,66 +46,59 @@ OPENOCD_TRANSPORT ?= hla_swd
 OPENOCD_TARGET = target/stm32f4x.cfg
 else ifeq ($(TREZOR_MODEL),$(filter $(TREZOR_MODEL),DISC2))
 MCU = STM32U5
-LAYOUT_FILE = embed/models/D002/model_D002.h
 OPENOCD_TARGET = target/stm32u5x.cfg
 else
 $(error Unknown TREZOR_MODEL: $(TREZOR_MODEL))
 endif
 
 
-FLASH_START              = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FLASH_START)
-BOARDLOADER_START        = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOARDLOADER_START)
-BOOTLOADER_START         = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOOTLOADER_START)
-FIRMWARE_START           = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_START)
-FIRMWARE_P2_START        = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_P2_START)
-STORAGE_1_OFFSET         = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} STORAGE_1_OFFSET)
-STORAGE_2_OFFSET         = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} STORAGE_2_OFFSET)
-STORAGE_SIZE             = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} NORCOW_SECTOR_SIZE)
-BOARDLOADER_MAXSIZE      = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOARDLOADER_IMAGE_MAXSIZE)
-BOOTLOADER_MAXSIZE       = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOOTLOADER_IMAGE_MAXSIZE)
-FIRMWARE_MAXSIZE         = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_IMAGE_MAXSIZE)
-FIRMWARE_P1_MAXSIZE      = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_P1_IMAGE_MAXSIZE)
-FIRMWARE_P2_MAXSIZE      = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_P2_IMAGE_MAXSIZE)
-BOARDLOADER_SECTOR_START = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOARDLOADER_SECTOR_START)
-BOARDLOADER_SECTOR_END   = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOARDLOADER_SECTOR_END)
-BOOTLOADER_SECTOR_START  = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOOTLOADER_SECTOR_START)
-BOOTLOADER_SECTOR_END    = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} BOOTLOADER_SECTOR_END)
-FIRMWARE_SECTOR_START    = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_SECTOR_START)
-FIRMWARE_SECTOR_END      = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_SECTOR_END)
-FIRMWARE_P2_SECTOR_START = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_P2_SECTOR_START)
-FIRMWARE_P2_SECTOR_END   = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} FIRMWARE_P2_SECTOR_END)
-STORAGE_1_SECTOR_START   = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} STORAGE_1_SECTOR_START)
-STORAGE_1_SECTOR_END     = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} STORAGE_1_SECTOR_END)
-STORAGE_2_SECTOR_START   = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} STORAGE_2_SECTOR_START)
-STORAGE_2_SECTOR_END     = $(shell python ./tools/layout_parser.py ${LAYOUT_FILE} STORAGE_2_SECTOR_END)
+FLASH_START              = $(shell layout_parser ${TREZOR_MODEL} FLASH_START)
```

Comment:
`FLASH_START` seems to be unused.

Reply:
i'd keep it around in case something like `make flash` would use it? wdyt @TychoVrahe